SPECS-EXTENDED/opencsd/opencsd.spec

```
@@ -0,0 +1,106 @@
+Summary:        An open source CoreSight(tm) Trace Decode library
+Name:           opencsd
+Version:        1.4.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/Linaro/OpenCSD
+Source:         https://github.com/Linaro/OpenCSD/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  git
+BuildRequires:  make
+BuildRequires:  patch
+
+%description
+This library provides an API suitable for the decode of ARM(r)
+CoreSight(tm) trace streams.
+
+%package devel
+Summary:        Development files for the CoreSight(tm) Trace Decode library
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+The opencsd-devel package contains headers and libraries needed
+to develop CoreSight(tm) trace decoders.
+
+%prep
+%setup -q -n OpenCSD-%{version}
+
+%build
+cd decoder/build/linux
+export CFLAGS="%{optflags}"
+export CXXFLAGS="%{optflags}"
+LIB_PATH=%{_lib} make %{?_smp_mflags}
+
+
+%install
+cd decoder/build/linux
+PREFIX=%{buildroot}%{_prefix} LIB_PATH=%{_lib} make install DISABLE_STATIC=1 DEF_SO_PERM=755
+
+
+%check
+# no upstream unit tests yet
+
+%files
+%license LICENSE
+%doc HOWTO.md README.md
+%{_libdir}/*so\.*
+%{_bindir}/*
+
+%files devel
+%doc decoder/docs/prog_guide/*
+%{_includedir}/*
+# no man files..
+%{_libdir}/*so
+
+%changelog
+* Wed Jul 12 2023 Saranya R <rsaranya@microsoft.com> - 2.28.3-1
```

I think it might be a copy-paste error:
```suggestion
* Wed Jul 12 2023 Saranya R <rsaranya@microsoft.com> - 1.4.0-1
```